model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -83,6 +83,10 @@ class LicenseInfoResolver(
                 licenseInfo.concludedLicenseInfo.concludedLicense?.also {
                     originalExpressions += ResolvedOriginalExpression(expression = it, source = LicenseSource.CONCLUDED)
                 }
+
```

Comment:
Unfortunately, I had a typo in my proposed commit message: "to be mapped to copyright" needs to say "to be mapped to copyright*s*".

Reply:
1 typo fixed